feat: refine readme (#51)
reearth/reearth-web: README.md

# reearth-web

> This is front-end repository of [Re:Earth](https://github.com/reearth/reearth)

Reviewer: ```suggestion
This is the front-end repository of [Re:Earth](https://github.com/reearth/reearth)
```

> ## Bug report and Feature requests

Reviewer: ```suggestion
## Bug reporting and requesting features
```

> If your have any requests for new features all are welcome! Just create [an issue under Re:Earth repository](https://github.com/reearth/reearth/issues)

Reviewer: ```suggestion
Any requests for new features are very welcome! Just create [an issue under Re:Earth repository](https://github.com/reearth/reearth/issues) and the core team will take it into consideration.
```

> When you found any bugs, please create bug issues from [bug report template](https://github.com/reearth/reearth/issues/new?assignees=&labels=&template=bug_report.md&title=).

Reviewer: ```suggestion
When you find a bug, please create a bug issue with the [bug report template](https://github.com/reearth/reearth/issues/new?assignees=&labels=&template=bug_report.md&title=).
```

> ## Discussions
> [Here](https://github.com/reearth/reearth/discussions) we have the discussion all topics.

Reviewer: ```suggestion
## Discussion
[Here](https://github.com/reearth/reearth/discussions) we have a dedicated area for discussions on all topics.
```
